lib/utils.pm

```diff
@@ -397,6 +398,23 @@ sub sle_version_at_least {
     die "unsupported SLE $version_variable $version in check";
 }
 
+sub install_to_other_at_least {
```

Comment:
I will add comment.

But I recommend to keep it in utils. Because after we introduce the parameter "INSTALL_TO_OTHERS", which will force to install the system to the version you give, instead of the current PRODUCT the test is put in. So theoretically, all future conditions which do  or not do something according to the version the system is in should use the REAL version it is , rather than the PRODUCT version the test is put.